bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/dbTools/timingwindowcheck/TimingWindowFunctor.scala

```diff
@@ -47,7 +47,7 @@ object TimingWindowFunctor {
 private class TimingWindowFunctor extends DBAbstractQueryFunctor {
   import TimingWindowFunctor.{ isActive, isScience }
 
-  val results: Buffer[(SPProgramID, List[(SPObservationID, Instant)])] = Buffer.empty
+  var results: Buffer[(SPProgramID, List[(SPObservationID, Instant)])] = Buffer.empty
```

Comment:
The idea with the `var` was to change to an immutable data type like `List`.  If you stick with `Buffer` you don't need `var`.